test/Microsoft.AspNetCore.Mvc.IntegrationTests/ActionParametersIntegrationTest.cs

```diff
@@ -562,5 +596,37 @@ System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator()
                 return GetEnumerator();
             }
         }
+
+        // By default the ComplexTypeModelBinder does not bind to abstract types as we cannot construct them,
+        // but a developer could change this behavior by overridng CreateModel
```

Comment:
The `ClassWithNoDefaultConstructor` class isn't `abstract`; comment should be about the lack of parameterless constructor.